src/aiortc/codecs/h264.py

```diff
@@ -225,12 +225,13 @@ def _split_bitstream(buf: bytes) -> Iterator[bytes]:
         # TODO: write in a more pytonic way,
         # translate from: https://github.com/aizvorski/h264bitstream/blob/master/h264_nal.c#L134
         i = 0
+        buflen = len(buf)
```

Comment:
The calls to `len` were taking a ton of time cumulatively. And `buf` never changes length, so they are unnecessary.

Reply:
I split this out into a separate pr on aiortc: https://github.com/aiortc/aiortc/pull/564

Reply:
I'll test this optimization in my side.

Reply:
About your CPU problem, check the `ffmpeg` process CPU consumption isolated first.

Reply:
Is it possible to do the same in `_packetize_stap_a` function?

Reply:
I added some more analysis to https://github.com/aiortc/aiortc/pull/564. Looks like a 42% improvement if I'm reading it right.

According to the profiling data `_packetize_stap_a` wasn't anywhere near the top of the most time consuming functions. So possibly you could do something similar, but it's not going to give much of a performance gain.